charts/du/values.yaml

```
@@ -65,6 +66,13 @@ serviceAccount:
   annotations: {}
   name: ""
 
+rbac:
+  enabled: true
+  rules:
+    - apiGroups: [""]
+      resources: ["services"]
+      verbs: ["get", "list"]
```

We don't need to list, so we can drop that. 

```suggestion
rbac:
  enabled: true
  rules:
    - apiGroups:
        - ""
      resources:
        - services
      verbs:
        - get
```